unbound/metadata.csv

```diff
@@ -0,0 +1,100 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
Any metrics described as `number of queries` should be updated to:
metrics_type - `count`
interval - Add the interval if known
unit_name - `query`
description - `The number of queries...`

Any metrics described as `number of <something>` should be updated to:
metrics_type - `count`
interval - Add the interval if known
description - `The number of <something>...`

Reply:
Took a crack at this in https://github.com/DataDog/integrations-extras/pull/451/commits/187abc7aa1d94e8e4acd0340799480156ba34203.